Snowflake.Data/Snowflake.Data.csproj

```diff
@@ -19,6 +19,7 @@
   </PropertyGroup>
   
   <ItemGroup>
+    <PackageReference Include="Apache.Arrow" Version="12.0.1" />

```

Comment:
Before merge should we have CASEC ticket reviewed, right?

Reply:
CASEC-8693 approved